plugins/kyverno/cmd/main.go

```diff
@@ -0,0 +1,161 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"os"
+
+	"github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/restmapper"
+	ctrl "sigs.k8s.io/controller-runtime"
+)
+
+type Client struct {
+	RestMapper       meta.RESTMapper
+	DynamicInterface dynamic.Interface
+}
+
+func main() {
+	logrus.Info("Starting Kyverno plugin")
+	client, err := getKubeClient()
+	if err != nil {
+		panic(err)
```

Comment:
Expected errors should not `panic`, as an alternative, use `logrus.Fatalf("error on doing xyz: %w, err)` instead.

Reply:
done